src/nsidc/metgen/metgen.py

```diff
@@ -653,7 +652,9 @@ def stage_files(configuration: config.Config, granule: Granule) -> Granule:
     """
     Stage a set of files for the Granule in S3.
     """
-    stuff = granule.data_filenames + [granule.ummg_filename] + granule.browse_filenames
+    stuff = concat(
```

Comment:
Ha, I meant to change this name...oops. I didn't know what to call it when I wrote the code. A little peek into my development habits.